setup.py

```diff
@@ -10,6 +12,16 @@
 if sys.version_info[0] < 3:
   sys.exit('ERROR: Python < 3 is unsupported.')
 
+platform.system()
```

Comment:
Heh, and since `print` isn't a statement in python3, this breaks for every case where this code is reachable (see https://travis-ci.org/google/vroom/jobs/95810106).